src/main/java/org/springframework/data/elasticsearch/annotations/InnerField.java

```diff
@@ -48,4 +49,6 @@
 	String analyzer() default "";
 
 	String normalizer() default "";
+
+	int ignoreAbove() default -1;
```

Comment:
I'd prefer to use an `Integer`here which as default value is set to `null`.

Reply:
But annotation does not support the wrap type.